src/renderers/dom/shared/dangerousStyleValue.js

```diff
@@ -42,6 +42,7 @@ function dangerousStyleValue(name, value, component) {
   }
 
   if (
+    name.indexOf('--') !== 0 &&
```

Comment:
It seems unfortunate to do this check here since we also check in the code path that calls this function. So it will get called twice for each property.

Can you please keep a single check in `setValueForStyles` (and any other methods calling `dangerousStyleValue`), and instead add a boolean `isCustomProperty` to `dangerousStyleValue`?